setup_robotino/robotino-adduser

```diff
@@ -35,6 +35,7 @@ if [ "$1" != "" ]; then
 	sudo -u $1 ssh $1@robotino 'exit'
 	echo "copying ssh-keys to other pcs, please enter password of new user"
 	sudo -u $1 cat /u/$1/.ssh/id_rsa.pub | ssh pi@raspberrypi "cat >> /u/$1/.ssh/authorized_keys"
```

Comment:
I think this is not necessary anymore as ssh-copy-id should remove the need to write the keys to .ssh/authorized_keys manually.